{
  try {
    _coll.clear();
  }
 catch (  MongoDBException e) {
    logger.error("Error while clearing store.",e);
  }
}
